packages/flutter/lib/src/services/system_navigator.dart

```diff
@@ -21,10 +21,14 @@ class SystemNavigator {
   /// `dismissViewControllerAnimated:completion:` if the top view
   /// controller is a `FlutterViewController`.
   ///
+  /// The optional `animated` parameter is ignored on all platforms
+  /// except iOS where it is an argument to the aforementioned
+  /// methods.
+  ///
   /// This method should be preferred over calling `dart:io`'s [exit]
   /// method, as the latter may cause the underlying platform to act
   /// as if the application had crashed.
-  static Future<void> pop() async {
-    await SystemChannels.platform.invokeMethod<void>('SystemNavigator.pop');
+  static Future<void> pop({bool animated}) async {
+    await SystemChannels.platform.invokeMethod<void>('SystemNavigator.pop', animated);
```

Comment:
Is this api backward compatible?

Reply:
Yep, it's an optional named parameter.

Reply:
Oh I mean if user use the channel directly without animated parameter
```
SystemChannels.platform.invokeMethod<void>('SystemNavigator.pop');
```
I think it will pass nil into https://github.com/flutter/engine/pull/12752/files#diff-39b3bbecd789ef05078fae67dfd78e0aR198
will that be a problem?

Reply:
it seems fine, it looks like it will default to false if no parameter passed into message channel